core/new-gui/src/index.html

```diff
@@ -9,10 +9,11 @@
   <link rel="icon" type="image/x-icon" href="favicon.ico">
 
   <!-- Add Angular Material Fonts -->
-  <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet"> -->
 
 </head>
 <body>
+  <script async defer src="https://apis.google.com/js/api.js"></script>
```

Comment:
it's not good to add async script in body. can we move it to `header` or `footer` instead? also I think the better way to do is add it to angular packages so that it can be used inside corresponding .ts file.

Reply:
moved to `<head>`.